File: C/prob_6.c
#include<stdio.h>
#include<math.h>
int main()
{
    int sum1 = 0, sum2 = 0, diff, temp = 0;
    for(int i=1; i<=100; i++)
    {
        int temp2 = pow(i, 2);
        sum1 += temp2;
    }
    for(int i=1; i<=100; i++)
    {
        temp += i;
    }
    int temp3 = pow(temp, 2);
    sum2 += temp3;
    diff = sum2 - sum1;
    printf("%d", diff);
    return 0;
}

File: Python2/prob_6.py
import math
sum1 = 0
for i in range(1,101):
    sum1 += math.pow(i, 2)

temp = 0
sum2 = 0
for j in range(1, 101):
    temp += j

sum2 =  math.pow(temp, 2)

print(sum2 - sum1)